python/cuml/svm/svc.pyx

```diff
@@ -0,0 +1,282 @@
+        support_vectors_ : float, shape [n_support, n_cols]
+            Device array of support vectors
+        dual_coef_ : float, shape = [1, n_support]
+            Device array of coefficients for support vectors
+        intercept_ : int
+            The constant in the decision function
+        fit_status_ : int
+            0 if SVM is correctly fitted
+        coef_ : float, shape [1, n_cols]
+            Only available for linear kernels. It is the normal of the
+            hyperplane.
+            coef_ = sum_k=1..n_support dual_coef_[k] * support_vectors[k,:]
+
+        For additional docs, see `scikitlearn's SVC
+        <https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html>`_.
+        """
+        super(SVC, self).__init__(handle, C, kernel, degree, gamma, coef0, tol,
+                                  cache_size, max_iter, nochange_steps,
+                                  verbose)
+        self.svmType = C_SVC
+
+    def fit(self, X, y):
+        """
+        Fit the model with X and y.
+
+        Parameters
+        ----------
+        X : array-like (device or host) shape = (n_samples, n_features)
+            Dense matrix (floats or doubles) of shape (n_samples, n_features).
+            Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
+            ndarray, cuda array interface compliant array like CuPy
+
+        y : array-like (device or host) shape = (n_samples, 1)
+            Dense vector (floats or doubles) of shape (n_samples, 1).
+            Acceptable formats: cuDF Series, NumPy ndarray, Numba device
+            ndarray, cuda array interface compliant array like CuPy
+
+        """
+
+        cdef uintptr_t X_ptr, y_ptr
+
+        X_m, X_ptr, self.n_rows, self.n_cols, self.dtype = \
+            input_to_dev_array(X, order='F')
+
+        y_m, y_ptr, _, _, _ = input_to_dev_array(y,
+                                                 convert_to_dtype=self.dtype)
+
+        self._dealloc()  # delete any previously fitted model
+        self._coef_ = None
+
+        cdef KernelParams _kernel_params = self._get_kernel_params(X_m)
+        cdef svmParameter param = self._get_svm_params()
+        cdef svmModel[float] *model_f
+        cdef svmModel[double] *model_d
+        cdef cumlHandle* handle_ = <cumlHandle*><size_t>self.handle.getHandle()
+
+        if self.dtype == np.float32:
+            model_f = new svmModel[float]()
+            svcFit(handle_[0], <float*>X_ptr, <int>self.n_rows,
+                   <int>self.n_cols, <float*>y_ptr, param, _kernel_params,
+                   model_f[0])
+            self._model = <uintptr_t>model_f
+        elif self.dtype == np.float64:
+            model_d = new svmModel[double]()
+            svcFit(handle_[0], <double*>X_ptr, <int>self.n_rows,
+                   <int>self.n_cols, <double*>y_ptr, param, _kernel_params,
+                   model_d[0])
+            self._model = <uintptr_t>model_d
```

Comment:
If I’m not mistaken, this pointer will make the model not pickleable. Dealing with this can be a follow up PR, could you open an issue?